packages/docs/themes/nimatron/source/css/nimatron.scss

```diff
@@ -100,33 +102,34 @@
     display: grid;
     grid-template-columns: minmax(auto, ($max-line-length * 2)) minmax(auto, auto);
     min-width: 0;
-    box-shadow: 0 2px ms(0) rgba(0, 0, 0, 0.2);
+    box-shadow: 0 2px $size-base rgba(0, 0, 0, 0.2);
   }
 
-  h2 {
+  h2:not([class]) {
     position: relative;
-    padding-top: $base-spacing;
+    // padding-top: $spacing-m;
```

Comment:
can we remove the commented out style here please? If we're not already, using something like [stylelint](https://stylelint.io/) will help us avoid checking in this type of cruft into version control.

Reply:
removed! We do use stylelint in this project, I think this is a good thing to add to our config. Any objection @edburyenegren-okta ?